pages/linux/hwinfo.md

```diff
@@ -3,17 +3,17 @@
 > Probe for the hardware present in the system.
 > More information: <https://manpages.opensuse.org/hwinfo/hwinfo.8.en.html>.
 
-- Get graphics card information:
+- Display all available hardware information:
 
-`hwinfo --gfxcard`
+`hwinfo`
 
-- Get network device information:
+- Display information about a specific hardware component:
 
-`hwinfo --network`
+`hwinfo --{{cpu|memory|disk|gfxcard|network|usb|pci|keyboard|mouse|monitor|sound|fingerprint}}`
```

Comment:
There are many more options but I believe these are the most useful

Any better ideas on how to tell the user about them?

Reply:
```suggestion
`hwinfo --{{cpu|memory|disk|gfxcard|network|usb|pci|keyboard|mouse|monitor|sound|fingerprint|...}}`
```
Like this